apps/newsletters-ui/src/app/components/SchemaForm/SchemaField.tsx

```diff
@@ -39,13 +48,36 @@ interface SchemaFieldProps<T extends z.ZodRawShape> {
 	maxOptionsForRadioButtons: number;
 }
 
-const WrongTypeMessage = (props: { field: FieldDef }) => (
+const WrongValueTypeMessage = (props: { field: FieldDef }) => (
 	<div>
-		wrong value type ({fieldValueAsDisplayString(props.field)}) for{' '}
-		{props.field.type}
+		<p>WRONG VALUE TYPE FOR {props.field.key}</p>
```

Comment:
```suggestion
		<p>Wrong value type for <strong>{props.field.key}</strong></p>
```

Perhaps less shouty? Caps make me anxious 😨